openvdb/openvdb/tree/LeafBuffer.h

```diff
@@ -330,7 +330,7 @@ template<typename T, Index Log2Dim>
 inline bool
 LeafBuffer<T, Log2Dim>::deallocate()
 {
-    if (mData != nullptr && !this->isOutOfCore()) {
```

Comment:
This is not the same as we will return false for nullptr and in-core in the previous  code, but now are returning true..... 

I'm not sure of the implications though.

Reply:
If the implication is unknown I'll change it back thanks for reviewing.